bears/general/CasingBear.py

```diff
@@ -0,0 +1,153 @@
+import re
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.bearlib.naming_conventions import (
+    to_camelcase, to_pascalcase, to_snakecase)
+from bears.general.AnnotationBear import AnnotationBear
+from coalib.bearlib.languages.LanguageDefinition import LanguageDefinition
+from coalib.results.SourceRange import SourceRange
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+
+
+class CasingBear(LocalBear):
+
+    def run(self,
+            filename,
+            file,
+            dependency_results,
+            casing: str,
+            language: str):
```

Comment:
you probably wanna add an optional param for external coalang like @abhsag24 did for his bear